sdk/sqlmanagement/Azure.ResourceManager.Sql/CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # Release History
 
-## 1.0.0-beta.2 (Unreleased)
+## 1.0.0-beta.2 (2021-12-28)
```

Comment:
please update the version in readme.md

Reply:
The version is automatically updated right after the release of a previous version and change logs are already updated by Dapeng.